packages/app/src/App.tsx

```diff
@@ -1,29 +1,29 @@
 import { FC } from "react";
 import { Route, Switch } from "react-router";
+import { Box, Grid } from "grommet";
 import { BrowserRouter as Router } from "react-router-dom";
 
-import TestnetBanner from "./components/testnet-banner";
 import Home from "./pages/home";
 import Token from "./pages/token";
 import { NotificationsList } from "./components/transactions";
 import Foot from "./components/footer";
-import { Grid } from "grommet";
+import Navbar from "./components/nav";
```

Comment:
The navbar doesn't need to be on the landing page only in the app

Reply:
I moved the banner to the navbar, and we are only showing that on the main page